### ISSUE TITLE
Add more textual information

### ISSUE THREAD

**Issue by tcrass** [SourceryOne/Grenar#3]
Hi there,

"Branches" (aka "Bernoulli Gate") was one of my favorite modules when playing around with the fabulous Cardinal eurorack emulator. Now, since I decided to go "real life" and assemble a true hardware modular synthesizer, I found it incredibly difficult to find a supplier for an MI Branches clone (now since Émilie ceased selling her genuine modules just when I discovered the miraculous world of modular synthesis). All the happier I was when I found your DIY "Branches" project, and I just ordered a "Grenar" kit though Tindie.
However, the aliexpress link you provide in your GitHub's README doesn't unambiguously lead to a specific Arduino module that would work with your hex files; it rather redirects to a choice of products which, apart from featuring different USB connectors, appear to differ in -- at least to me --  somewhat incomprehensible way. I'd greatly appreciate if you could be a bit more specific in this regard.
Furthermore, in you video description of the project, you mention that you had to alter a specific "row" of  Émilie's original code in order to make it work with the particular Arduino board you used. I suppose it would be inspiring for the modular community if you published not just the compiled .hex files, but also the modified source code in your github repository. Not really a pressing matter, but on the long run it might help keeping the "Branches" idea alive!

Anyway, thanks for the time and effort you spent on "Grenar"! Can't wait for the kit to arrive...

Cheers --

tcrass

**Comment by mager33** [comment]
Yes, Emilie opted for open-source. If I understand the licence right, you have to go open-source, too. So, please put the source code here!

**Comment by SourceryOne** [comment]
Oh, right. Sorry I missed that. Unsure if I have the source code left, but will remake the file and upload it. BUT the only difference is that in the makefile I changed this row: 
MCU_NAME       = 88
to:
MCU_NAME       = 328
But again as I say in the video, these two MCUs are so similar so it probably doesnt make a difference. But since Emelie has them both in her repository I thought I might aswell use the correct one. But will add the file aswell.
